modules/ding_mobilesearch/ding_mobilesearch.module

```diff
@@ -0,0 +1,504 @@
+        if (!empty($info['settings']['allowed_values']) && is_array($info['settings']['allowed_values'])) {
+          $settings = reset($info['settings']['allowed_values']);
+          if (!empty($settings['vocabulary'])) {
+            $vocabulary_machine_name = $settings['vocabulary'];
+          }
+        }
+        if ($vocabulary_machine_name) {
+          $vocabulary = taxonomy_vocabulary_machine_name_load($vocabulary_machine_name);
+          $vocabulary = $vocabulary->name;
+        }
+        else {
+          $vocabulary = $field['label'];
+        }
+        $items = field_get_items('node', $node, $machine_name);
+        $mapping['taxonomy'][$machine_name] = array(
+          'name' => $vocabulary,
+          'terms' => array(),
+        );
+        if (is_array($items)) {
+          foreach ($items as $row) {
+            if (!empty($row['tid'])) {
+              $term = taxonomy_term_load($row['tid']);
+              $mapping['taxonomy'][$machine_name]['terms'][] = $term->name;
+            }
+          }
+        }
+        if (empty($mapping['taxonomy'][$machine_name]['terms'])) {
+          unset($mapping['taxonomy'][$machine_name]);
+        }
+
+        if ($info['type'] == 'entityreference') {
+          $mapping['fields'][$machine_name] = array(
+            'name' => $field['label'],
+            'value' => array(),
+            'attr' => array(),
+          );
+          foreach ($items as $item) {
+            $mapping['fields'][$machine_name]['value'][] = $item['target_id'];
+          }
+        }
+        break;
+
+      case 'ting_reference_advanced':
+        $items = field_get_items('node', $node, $machine_name);
+        $mapping['fields'][$machine_name] = array(
+          'name' => $field['label'],
+          'value' => array(),
+          'attr' => array(),
+        );
+        foreach ($items as $item) {
+          $mapping['fields'][$machine_name]['value'][] = $item['ting_object_id'];
+        }
+        if (count($mapping['fields'][$machine_name]['value']) == 1) {
+          $mapping['fields'][$machine_name]['value'] = reset($mapping['fields'][$machine_name]['value']);
+        }
+        break;
+
+      case 'addressfield_standard':
+        $items = field_get_items('node', $node, $machine_name);
+        foreach ($items as $ikey => $row) {
+          foreach ($row as $key => $value) {
+            if ($key == 'element_key') {
+              continue;
+            }
+            $mapping['fields'][$machine_name . '__' . $key] = array(
+              'name' => $field['label'] . ' (' . str_replace('_', ' ', $key) . ')',
+              'value' => array(),
+              'attr' => array(),
+            );
+            $mapping['fields'][$machine_name . '__' . $key]['value'][$ikey] = $value;
+          }
+        }
+        break;
+    }
+  }
+
+  $module_enabled = system_list('module_enabled');
+  if (isset($module_enabled['opening_hours'])) {
+    $result = db_select('opening_hours', 'oh')
```

Comment:
This would yield a lot of data from the opening hours table. Maybe limit it to feature opening hours?

Also remember that some libraries uses the category option in opening hours to define different types of opening hours, which should be exported as well!